src/Service/PublicOffer/PublicOfferService.Enum.ts

```diff
@@ -0,0 +1,66 @@
+import { AxiosInstance } from "axios";
+import { PagedResult } from "./Data/Response";
+
+export class Enum {
+  _client: AxiosInstance;
+  constructor(client: AxiosInstance) {
+    this._client = client;
+  }
+
+  /**
+   * Read all operators
+   * @name cfg
+   * @param page The requested page
+   * @param pageSize The results to be queried per page
+   * @param operatorFilter The operator filter
+   * @param sort Sort by
+   * Returns Paged Result of Operators
+   */
+  GetOperators(cfg: { page: number, pageSize: number, operatorFilter: string | null, sort: string[] | null }) {
+    if (cfg.page < 1 || cfg.pageSize < 1){
+        throw new Error("Page and Page number need to be greater than 0. Page:" + cfg.page + " Page Size:" + cfg.pageSize);

```

Comment:
Return a rejected promise instead. So user can handle both errors in a promise catch